packages/salesforcedx-vscode-soql/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 50.17.0 - February 11, 2021
+
+## Added
+
+- Extract comments out of SOQL text before running query ([PR #2916](https://github.com/forcedotcom/salesforcedx-vscode/pull/2916))
```

Comment:
This should be "Add support for line comments at the top of SOQL files".

Reply:
Yep, much better to understand than the actual commit message.  I will finesse these in the future.  
I took out the work "Add" since it's in the "Added" section.